samples/excel/85-preview-apis/events-workbook-activated.yaml

```
@@ -0,0 +1,80 @@
+order: 5
+id: excel-events-workbook-activated
+name: Workbook activated event
+description: This sample shows how to register a workbook activated event handler.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+  content: |
+    $("#register-event-handler").click(() => tryCatch(registerEventHandler));
+
+    async function registerEventHandler() {
+      await Excel.run(async (context) => {
+        const workbook = context.workbook;
+
+        // Register the workbook activated event handler.
+        workbook.onActivated.add(workbookActivated);
+
+        await context.sync();
+        console.log("Added event handler for workbook activated.");
+      });
+    }
+
+    async function workbookActivated(event: Excel.WorkbookActivatedEventArgs) {
+      await Excel.run(async (context) => {
+        // Script that runs when the workbook is activated.
```

Let's avoid the term "script", since it's overloaded as is. Also, consider pulling this comment to above the `workbookActivated` function.
```suggestion
        // Callback function for when the workbook is activated.
```